modules/apps/frontend-taglib/frontend-taglib-sample-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,10 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<liferay-util:include page="/partials/search_paginator.jsp" servletContext="<%= application %>" />
```

Comment:
Fair enough, we can implement navigation with the second tag. Looks a bit silly on it's own, but I suppose it's fine for now.

Reply:
Done!